praw/models/util.py

```diff
@@ -44,6 +49,7 @@ def stream_generator(
     exclude_before: bool = False,
     pause_after: int | None = None,
     skip_existing: bool = False,
+    start_after: str | None = None,
```

Comment:
I'm not sure I like the name of this parameter. My concern is that it will cause confusion as `pause_after` is named similar and involves time and one could infer that `start_after` also involves time.

Reply:
What would you suggest? Originally I just had it as `before_attribute` but that wasn't clear either. Maybe `start_after_value`? Or `last_seen`?

Reply:
What about `continuation_id`

Reply:
Could work, but doesn't match the tense of the other parameters. How about `continue_from_id` / `continue_after_id` / `start_after_id`?

Reply:
Let's do `continue_after_id`